js/modules/k6/experimental/tracing/options.go

```diff
@@ -0,0 +1,40 @@
+package tracing
+
+import (
+	"errors"
+	"fmt"
+)
+
+// options are the options that can be passed to the
+// tracing.instrumentHTTP() method.
+type options struct {
+	// Propagation is the propagation format to use for the tracer.
+	Propagator string `js:"propagator"`
+
+	// Sampling is the sampling rate to use for the tracer.
+	Sampling *float64 `js:"sampling"`
+
+	// Baggage is a map of baggage items to add to the tracer.
+	Baggage map[string]string `js:"baggage"`
+}
+
+func (i *options) validate() error {
+	var (
+		isW3C    = i.Propagator == W3CPropagatorName
+		isB3     = i.Propagator == B3PropagatorName
+		isJaeger = i.Propagator == JaegerPropagatorName
+	)
+	if !isW3C && !isB3 && !isJaeger {
```

Comment:
Is there a reason we just don't do the equal comparision directly in the `if` sttatement?

Reply:
It's a style recommendation I discovered in the Google style guide, which, I find, helps with readability, as the `if` statement ends up much shorter and reads almost like a sentence. I'm not hung onto this though, if you have a strong opinion against that, I'm happy to drop it 🙇🏻